spec/models/embeddable/labbook_spec.rb

```diff
@@ -39,4 +39,34 @@
       end
     end
   end
+  describe 'its interactive' do
+    let(:args)    { {} }
+    let(:labbook) { Embeddable::Labbook.new args }
+    describe 'when it is not associated with an interactive' do
+      it 'should have no interactive' do
+        expect(labbook.interactive).to eql(nil)
+      end
+      it 'should be marked as disconnected' do
+        expect(labbook.is_connected?).to eql(false)
+      end
+
+      it 'it should not show up in the runtime' do
+        expect(labbook.show_in_runtime?).to eql(false)
+      end
+    end
+    describe 'when it has an interactive' do
+      let(:interactive) { MwInteractive.new  }
+      let(:args) { {interactive: interactive}}
```

Comment:
Yes. This is one of my favorite features of how lazy let evaluation works in specs.